DOCS-9484 Endpoints edits; Kim round 1 (#94)
mulesoft/docs-partner-manager: modules/ROOT/pages/endpoints.adoc

Partner Manager uses endpoints to:

* Receive transmissions from partners
* Receive application messages from the backend system
* Specify the final destination of the message
* Define transport protocols
* Specify where to send acknowledgments

== Endpoint Ownership

> 

Reviewer: Hi Emilee!

You will get used to this, but just to explain...: 

I am not only an editor-editor but also a reader-advocate editor who always looks for the simplest wording...or the simplest wording to ME, anyway, as someone coming to the text for the very first time. 

Your audience is likely to be more technical than I am, but I will err on the side of caution and make a lot of suggestions, just to make sure.

These are just suggestions. You are absolutely the owner of your topics and The Decider of the final wording, based on your superior understanding of both what you intend to say and who is consuming what you say.

But do please take a moment to consider the alternatives I offer as part of Content and Usability (CU) edits.

Copyedits are a different matter; you should follow them or explain to me why you think I am wrong, if you thing I am wrong.  ;-)

We can discuss this further and customize our working relationship to what works best for you as we go along. I am here first for the audience, but first-and-a-half for YOU, to make you life as easy as possible.

> Partner Manager enables you to assign the host or a particular partner as the owner of endpoints that are partner-facing (receive or send) to control the usage of endpoints within inbound and outbound message flows.

Reviewer: ```suggestion
Partner Manager enables you to assign the host or a particular partner as the owner of endpoints that are partner-facing (receive or send) to control the use of endpoints within inbound and outbound message flows.
```
I've had to read this a couple of times to understand this properly, and I am still struggling to find a way to parse this sentence.
Can you think of a simpler wording?

Reviewer: Actually, the meaning became more clear to me after I read the second and third sentences, so I'm wondering if this meaning would be more immediately clear in a presentation similar to this:

```suggestion
Partner Manager enables you to assign the ownership of inbound and outbound message flow endpoints to either:
* The host
+
Endpoints are available for use by all trading partners.
* A specific partner
+
Endpoints are available for use by that partner only.
```

Reviewer: Hmm... I still don't like that. 

It's just that there are several separate relationship details here, and putting more than one of the following into the same sentence confuses me a little:

1.  The beginnings and the endings of incoming and outgoing message flows between partners are called _partner-facing endpoints_.

(Are there any other types of endpoints; isn't any beginning or ending "customer facing"?)

2.  Partner Manager enables you to control how customer-facing endpoints are used by enabling you to designate (customer-facing) endpoints as either _host-owned_ or _partner-owned_.

(Using Partner Manager, you control how an endpoint is used by making either the host or a specific partner the endpoint owner.)

3.  Host-owned (customer-facing) endpoints can be used by all partners; partner-owned (customer-facing) endpoints can be used by only that partner that owns them.

(If you assign the host as owner.... If you assign a partner as owner...)

Reviewer: Maybe a graphic?

> Endpoints owned by the host are available for selection in message flows for any trading partners. Endpoints owned by a specific trading partner are available for selection in message flows created for that partner only.

Reviewer: "...are available for selection in message flows for any" is both passive and I believe ungrammatical: 

"...are available for selection BY any partners...."
"...are available TO any partners"

Again, I think this could be simpler:

```suggestion
Any trading partner can use an endpoint owned by the host ["in message flows" if nec.], but only the owning partner can use a partner-owned endpoint.
```
